packages/manager/src/features/Account/AccountLoginsTableRow.tsx

```diff
@@ -0,0 +1,27 @@
+import { AccountLogin } from '@linode/api-v4/lib/account/types';
+import * as React from 'react';
+import Hidden from 'src/components/core/Hidden';
+import TableCell from 'src/components/TableCell';
+import TableRow from 'src/components/TableRow';
+import formatDate from 'src/utilities/formatDate';
+
+const AccountLoginsTableRow: React.FC<AccountLogin> = (props) => {
```

Comment:
Consider moving away from `React.FC` on a component like this because `React.FC` implies that the component takes children. 

```suggestion
const AccountLoginsTableRow = (props: AccountLogin) => {
```